packages/@react-aria/dnd/src/useDroppableCollection.ts

```diff
@@ -68,11 +68,6 @@ export function useDroppableCollection(props: DroppableCollectionOptions, state:
         return 'cancel';
       }
 
-      if (state.isDropTarget(target)) {
```

Comment:
note to any other reviewers, this section of code appeared to be short-circuiting us. I'm not sure why it existed other than potentially as an optimization, it doesn't get the Drop Operation, so it disregards trying the root if the operation ends up being cancel, the remaining code below should always return the correct information.